test/puppetlabs/trapperkeeper/services/status/status_core_test.clj

```diff
@@ -124,10 +124,6 @@
           (is (every? #(or (< 0 %) (= -1 %)) (vals (:non-heap-memory jvm-metrics))))
           (is (= #{:max :used} (ks/keyset (:file-descriptors jvm-metrics))))
           (is (every? #(< 0 %) (vals (:file-descriptors jvm-metrics))))
-          (is (every? #(= #{:count :total-time-ms} (ks/keyset %))
-                      (vals (:gc-stats jvm-metrics))))
-          (is (every? #(<= 0 %)                             ;; Possible that no major collections occurred.
-                      (mapcat #(vals %)
-                              (vals (:gc-stats jvm-metrics)))))
+          (is (contains? jvm-metrics :gc-stats))
```

Comment:
I loosened these tests as Schema is doing a good job of enforcing assertions about the structure and data types of the response. Re-writing the same assertions in tests is a lot of effort just to produce a fragile duplication of checks which already exist.